vms/platformvm/block/executor/verifier.go

```diff
@@ -85,7 +85,18 @@ func (v *verifier) BanffProposalBlock(b *block.BanffProposalBlock) error {
 	onAbortState.SetTimestamp(nextChainTime)
 	changes.Apply(onAbortState)
 
-	return v.proposalBlock(&b.ApricotProposalBlock, onCommitState, onAbortState)
+	// Apply the changes, if any, from processing the decision txs.
+	onDecisionState, err := wrapState(onCommitState)
```

Comment:
Does it matter that `onDecisionState` wraps `onCommitState` when applying in `abort`?

Reply:
Nope, at this point onCommitState and onAbortState are the same.

Reply:
Good thing comments are free and you have room to elaborate on this 👀

Reply:
added a comment